### DISCUSSION TITLE
suggestion: mark cbegin and cend() as const functions

### DISCUSSION THREAD

**Discussion by ghost** [danielaparker/jsoncons#480] [Ideas]
in: include/jsoncons/basic_json.hpp
```
        const_iterator cbegin()
        {
            return first_;
        }
        const_iterator cend()
        {
            return last_;
        }
```
can be replaced with:
```
        const_iterator cbegin() const
        {
            return first_;
        }
        const_iterator cend() const
        {
            return last_;
        }
```

**Comment by danielaparker** [comment]
You're right, thanks. Would you care to make a pull request?

**Reply by ghost** [reply]
Im trying but facing this error:
Pull request creation failed. Validation failed: must be a collaborator
will try again in some hours
thanks

**Reply by amassalha** [reply]
@danielaparker I opened a pull request from my other account :
https://github.com/danielaparker/jsoncons/pull/481
